.github/workflows/appimage.yml

```diff
@@ -72,15 +85,53 @@ jobs:
       run: |
         sudo -u testbot bash -lc 'cd rdk/services/slam/builtin && sudo go test -v -run TestCartographerIntegration'
```

Comment:
I know not technically your change, but there's no reason to use sudo inside the bash command string. Go tests shouldn't need to be root.

Reply:
slam integration tests need sudo access: https://github.com/viamrobotics/viam-cartographer/actions/runs/4756567063/jobs/8452292353?pr=51

Reply:
Note: motion-benchmarks.yml does this as well in RDK

Reply:
I'm in the process of removing those cases in RDK, as they should not be doing that. One of the reasons we've got the "sudo -Hu testbot bash -lc ..." pattern everywhere is it forces thing to run as a normal user, and make thing break if they try to access stuff outside of where we expect and intend. If sudo is required here, then something else is wrong, and setup needs to be done differently. I expect the RDK code being checked out isn't being chown'd to the testbot user.

Reply:
Yeah, chown the rdk code to testboth first, before switching to the testbot user for the actual test run.